input/listen.go

```diff
@@ -6,42 +6,61 @@ import (
 )
 
 func listen(addr string, handler Handler) error {
-	laddr, err := net.ResolveTCPAddr("tcp", addr)
```

Comment:
IIRC there was a good reason to do the resolve in a separate step, but i don't remember what it was.
you may have to dig through the history to find out

Reply:
never mind. i see now this is part of `listenTcp`